src/bin/miri.rs

```diff
@@ -315,7 +315,9 @@ fn main() {
         } else if arg == "-Zmiri-disable-stacked-borrows" {
             miri_config.borrow_tracker = None;
         } else if arg == "-Zmiri-tree-borrows" {
-            miri_config.borrow_tracker = Some(BorrowTrackerMethod::TreeBorrows);
+            if miri_config.borrow_tracker.is_some() {
+                miri_config.borrow_tracker = Some(BorrowTrackerMethod::TreeBorrows);
+            }
```

Comment:
I think this should just always set borrow_tracker.

The docs should explain that disable-stacked-borrows disables both of them (but later flags take precedence)